Project/DAQ_System/Program/Src/Platform/I2C/STM/i2cHalWrapper_stmf4.hpp

```diff
@@ -15,20 +15,16 @@ namespace platform{
 
 class i2cHalWrapperStmf4 : public II2C{
 public:
-	i2cHalWrapperStmf4(I2C_HandleTypeDef &hi2c, uint8_t SAD);
+	i2cHalWrapperStmf4(I2C_HandleTypeDef &hi2c);
 
 	virtual ~i2cHalWrapperStmf4();
 
-	virtual void Master_Receive(uint8_t*) override;
+	virtual void Receive(uint8_t*, uint8_t, uint8_t) override;
 
-	virtual void Master_Transmit(uint8_t*) override;
-
-	virtual void SetResponse(uint8_t) override;
+	virtual void Transmit(uint8_t*, uint8_t) override;
 
 private:
 	I2C_HandleTypeDef bx_i2c_;
```

Comment:
You can remove "bx" from `bx_i2c_`.

"bx" comes from ST's CAN controller being formally named `BxCAN`.